lib/autoyast.pm

```diff
@@ -23,6 +28,13 @@ use registration qw(scc_version get_addon_fullname);
 
 our @EXPORT = qw(expand_template init_autoyast_profile);
 
+=head2 expand_patterns
+
+ expand_patterns();
+
+Expand patterns for sle12 and sle15
```

Comment:
Please give some info about the return value.
I would not really know what this function does from your description.